### ISSUE TITLE
Add guides for non-admin users (end-users)

### ISSUE THREAD

**Issue by ptgott** [gravitational/teleport#10050]
## Details
Our guides tend to focus on users who are setting up Teleport on their organization's infrastructure. Let's plan to write one or more guides for users who have joined an organization that accesses infra via Teleport, but are not involved in configuring or maintaining their Teleport deployment. We'll need to figure out a good way to make this content visible from the home page of the docs site.

Depending on how much content we'll be adding for non-admin users, we can consider creating a new issue for each page we plan to write and either tracking these projects here or closing this issue once the planning is complete.

### Category
- Improve Existing

**Comment by ptgott** [comment]
We have now added a section of the docs on using Teleport's client tools.
